src/lib/components/Editor/Editors/ArticleEditor.tsx

```diff
@@ -1,6 +1,7 @@
 import { Container } from '@material-ui/core';
```

Comment:
ArticleEditor and NoteEditor are very similar. can you think of how can we get rid of code duplication here?